docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.horizontal-card.scss

```diff
@@ -0,0 +1,144 @@
+.horizontal-card {
```

Comment:
Have we discussed using prefixes for classes? 

- `.obj-` for objects/layout
- `.cmp-` for components
- `.util-` for utility classes

This would come in handy when debugging and writing documentation and will also help distinguish our styles from Decanters.

Reply:
yeah I keep wanting to use hb for humsci_basic and then we could do:
- `.hb-cmp_`
- `.hc-obj_`

thoughts? @bryanbraun @kaseybon @nrambeck

Reply:
I like it

Reply:
variables: `.hb`

Reply:
done! I ended up using hb_ in most places without always namespacing the type of ITCSS thing